local-tests/test_catch_up.py

```diff
@@ -0,0 +1,75 @@
+#!/bin/env python
+import os
+import sys
+from os.path import abspath, join
+from time import sleep
+
+from chainrunner import Chain, Seq, generate_keys
+
+# Path to working directory, where chainspec, logs and nodes' dbs are written:
+workdir = abspath(os.getenv('WORKDIR', '/tmp/workdir'))
+# Path to the aleph-node binary (important use short-session feature):
+binary = abspath(os.getenv('ALEPH_NODE_BINARY', join(workdir, 'aleph-node')))
+
+
+def check_finalized(nodes):
+    results = [node.highest_block() for node in nodes]
+    highest, finalized = zip(*results)
+    print('Blocks seen by nodes:')
+    print('  Highest:   ', *highest)
+    print('  Finalized: ', *finalized)
+
+    return finalized
```

Comment:
This is almost 1:1 duplicate from the other test. Maybe moving it to `chainrunner/utils.py` is not a bad idea?

Reply:
done